system/modules/timelog/partials/templates/timelogwidget.tpl.php

```diff
@@ -36,20 +37,20 @@
                                                 "id|name"		=> "start_time",
                                                 "value"			=> !empty($active_log) ? $active_log->getTimeStart() : null,
                                                 "pattern"		=> "^(0?[0-9]|1[0-9]|2[0-3]):[0-5][0-9](\s+)?(AM|PM|am|pm)?$",
-                                                "placeholder"	=> "12hr format: 11:30pm or 24hr format: 23:30",
-                                                "required"		=> "true"
+                                                "placeholder"	=> "12hr format: 11:30pm or 24hr format: 23:30"
                                         ])); ?>
 				</label>
 			</div>
 		</div>
                 <br/>
 		<div class="row">
 			<div class="large-12 columns">
-				<button class="button" onclick="saveTimer();">Start</button>
+				<input class="button" type="submit" value="Save">
```

Comment:
You dont need to change the button to an input... also "type="submit"" is assumed if you omit the type. The input tag should also have the closing slash after it, i.e. "/>" instead of ">"